src/components/Dialogs/Alert/__tests__/__snapshots__/Alert.spec.tsx.snap

```diff
@@ -1,430 +1,5 @@
-                  "left": "40px",
-                  "outline": "none",
-                  "overflow": "auto",
-                  "padding": "20px",
-                  "position": "absolute",
-                  "right": "40px",
-                  "top": "40px",
-                },
-                "overlay": Object {
-                  "backgroundColor": "rgba(255, 255, 255, 0.75)",
-                  "bottom": 0,
-                  "left": 0,
-                  "position": "fixed",
-                  "right": 0,
-                  "top": 0,
-                },
-              }
-            }
-            isOpen={true}
-            onAfterOpen={[Function]}
-            overlayClassName={
-              Object {
-                "afterOpen": "Modal__overlay--entered",
-                "base": "Modal__overlay",
-                "beforeClose": "Modal__overlay--exited",
-              }
-            }
-            overlayRef={[Function]}
-            parentSelector={[Function]}
-            portalClassName="ReactModalPortal"
-            role="dialog"
-            shouldCloseOnEsc={true}
-            shouldCloseOnOverlayClick={true}
-            shouldFocusAfterRender={true}
-            shouldReturnFocusAfterClose={true}
-            style={
-              Object {
-                "content": Object {},
-                "overlay": Object {},
-              }
-            }
-          >
-            <div
-              className="Modal__overlay Modal__overlay--entered"
-              onClick={[Function]}
-              onMouseDown={[Function]}
-              style={Object {}}
-            >
-              <div
-                aria-label="Content label"
-                className="Modal__content Dialog Modal__content--entered"
-                onClick={[Function]}
-                onKeyDown={[Function]}
-                onMouseDown={[Function]}
-                onMouseUp={[Function]}
-                role="dialog"
-                style={Object {}}
-                tabIndex="-1"
-              >
-                <div
-                  className="Dialog__content"
-                  role="alert"
-                >
-                  Body of the alert
-                </div>
-                <div
-                  className="Dialog__actions"
-                >
-                  <Button
-                    isPrimary={true}
-                    onClick={[MockFunction]}
-                  >
-                    <button
-                      className="Button Button--isPrimary"
-                      disabled={false}
-                      onClick={[MockFunction]}
-                      type="button"
-                    >
-                      OK
-                    </button>
-                  </Button>
-                </div>
-              </div>
-            </div>
-          </ModalPortal>
-        </Portal>
-      </Modal>
-    </Modal>
-  </Dialog>
-</Alert>
-`;
+exports[`Alert should render correctly 1`] = `<div />`;
```

Comment:
This doesn't look like the DOM we want to render, is it?

Reply:
:+1: